src/unity/python/turicreate/visualization/_plot.py

```diff
@@ -34,7 +34,7 @@ def _focus_client_app():
             delay .5
             tell application \"Turi Create Visualization\" to activate
             '''
-    focus = __subprocess._Popen(['osascript', '-'], stdout=_PIPE, stdin=_PIPE)
+    focus = _Popen(['osascript', '-'], stdout=_PIPE, stdin=_PIPE)
```

Comment:
This fix seems unrelated... are you building on a branch with multiple commits?

Reply:
Oh no, this was another bug I discovered in the focusing mechanism. It was a fix I made to my branch and never committed. It accidentally got included in this PR.

Reply:
Accidentally or intentionally? Do you intend for this change to go in with this PR, or should it be factored out into another PR?

Reply:
I accidentally added it. I'll refactor it out into another PR.

Reply:
Updated PR!